### PR TITLE
feat: email link to phone number archive

### PR DESCRIPTION
**Pull request by st3v3nhunt**

### CODE REVIEW

**Review by markysoft — COMMENTED** [review]
Looks good, a couple of queries before approving.

File: EmailPhoneNumbers/index.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+const zipPath = path.join(__dirname, zipFilename)
+
+async function uploadFile (context) {
+  await blockBlobClient.uploadFile(zipPath, { blobHTTPHeaders: { blobContentType: 'application/zip' } })
+  context.log(`Uploaded file: ${zipFilename} to container: ${phoneNumbersContainer}.`)
+}
+
+async function getSasUrl (context) {
+  const expiresOn = new Date(Date.now())
```
[the remark this answers is not in the document]
Comment (st3v3nhunt) [reply EmailPhoneNumbers/index.js:30]: Yes. This is a bit of a fudge for testing. I couldn't get the mocking of the `Date` constructor working so I went with the easier option of mocking `Date.now()`.

File: EmailPhoneNumbers/index.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        const sasUrl = await getSasUrl(context)
+        await sendEmail(context, sasUrl)
+        resolve('resolved')
+      })
+      output.on('error', err => {
+        context.log(err)
+        reject(err)
+      })
+
+      zipFile(output, blobContents)
```
[the remark this answers is not in the document]
Comment (st3v3nhunt) [reply EmailPhoneNumbers/index.js:74]: Yes! Good spot.

**Review by markysoft — APPROVED** [review]
LGTM

**Pull request merged**